src/Microsoft.NET.Sdk.Aspire/WorkloadManifest.in.targets

```diff
@@ -5,4 +5,14 @@
     Condition=" ('$(IsAspireHost)' == 'true') and ('$(SkipAspireWorkloadManifest)' != 'true') "
   />
 
+  <Import
```

Comment:
nit: If everything needs to be conditioned on `('$(IsAspireHost)' == 'true') and ('$(SkipAspireWorkloadManifest)' != 'true')` then maybe the contents should be moved to a separate file, and that file be conditionally imported here.

Reply:
In this case, this is a standard pattern for WorkloadManifest targets; every pack SDK import is supposed to be conditioned as WorkloadManifets.targets is technically loaded for every project.

Reply:
That's true. Moving all the stuff to a separate file would mean that the condition can be used at import time itself, thus ensuring that it isn't needed for every element in the targets.
`<Import Project="WorkloadManifest.Aspire.targets" Condition="('$(IsAspireHost)' == 'true') and ('$(SkipAspireWorkloadManifest)' != 'true')" />`

This would also allow having separate files for different tfm like `WorkloadManifest.Aspire.net8.targets`, and being able to condition them in the top level targets file.

Just thoughts, nothing blocking.

Reply:
I’m honestly not sure whether a workload manifest pack can include files other than WorkloadManifest.json and WorkloadManifest.targets; I imagine it comes down to whether those particular files are hardcoded in any way in the Arcade build for workloads.

Reply:
This comes from the sdk, and that just looks for WorkloadManifest.targets . We can include other files next to it, and import into WorkloadManifest.targets . We used this style in `wasm-tools` workload - https://github.com/dotnet/runtime/blob/main/src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.Current.Manifest/WorkloadManifest.targets.in#L23

Reply:
Updated the targets to match this pattern

Reply:
> Condition=" ('$(IsAspireHost)' == 'true') and ('$(SkipAspireWorkloadManifest)' != 'true') "

Note that the parens in an expression like this are unnecessary.

Reply:
Removed the extra parenthesis